docs/maintainers/release-guide.md

```diff
@@ -0,0 +1,38 @@
+# Release Guide
+
+The target audience for this guide is maintainers of this repo. In general, the release process is as follows:
+
+1. Create a release candidate
+1. Test the release candidate
+    1. If the release candidate is not satisfactory, go back to step 1
+    1. If the release candidate is satisfactory, create an official release
+
+## Release Candidate
+
+To create a release candidate:
```

Comment:
I'd suggest convert it to a sub-header so that it can be referenced in the `Official Release` section.
```suggestion
### Creating a release candidate
```